### PR TITLE
Inline styles using cssText

### PR DESCRIPTION
**Pull request by frenzzy**
Same as #675 but using [`cssText`](https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration/cssText)

Demo: https://codepen.io/frenzzy/pen/PRaPrM/left/?editors=0010

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -157,13 +157,17 @@ export function app(state, actions, view, container) {
   function updateAttribute(element, name, value, oldValue, isSvg) {
     if (name === "key") {
     } else if (name === "style") {
```
Comment (jorgebucaran) [inline src/index.js]: @frenzzy Let's use `typeof value === "string"` instead (below) 🙏

File: src/index.js [hunk 2]
```diff
@@ -156,17 +156,21 @@ export function app(state, actions, view, container) {
...
     if (name === "key") {
-    } else if (name === "style") {
-      for (var i in clone(oldValue, value)) {
-        var style = value == null || value[i] == null ? "" : value[i]
-        if (i[0] === "-") {
-          element[name].setProperty(i, style)
+    } else {
+      if (name === "style") {
+        if (typeof value === "string") {
+          element[name].cssText = value
```
Comment (jorgebucaran) [inline src/index.js]: What if we also check if `value == null`? 🤔

Comment (frenzzy) [reply src/index.js]: maybe just `typeof value !== "object"` instead of `typeof value === "string" || value == null`?

Comment (frenzzy) [reply src/index.js]: or you are talking about `cssText = value == null ? '' : value`?
This could work too, but it seems it changes nothing

**Comment by jorgebucaran** [comment]
Thanks, @frenzzy. I am going to abandon this feature for the same reason given [here](https://github.com/facebook/react/issues/5397#issuecomment-153954732) as I hold the same opinion.

**Pull request closed without merging**